installer/roles/kubernetes/templates/deployment.yml.j2

```diff
@@ -28,7 +28,7 @@ metadata:
 spec:
   serviceName: {{ kubernetes_deployment_name }}
```

Comment:
`serviceName` is not in the kube `Deployment` spec schema so I think this needs to be removed to avoid an error.

Reply:
Weird. It seemed to work for me, but will look into it. Thanks for testing this out 😄